lib/fluent/plugin/parser_syslog.rb

```diff
@@ -48,6 +48,8 @@ def configure(conf)
         super
 
         @time_parser_rfc3164 = @time_parser_rfc5424 = nil
+        @time_parser_rfc5424_without_secfrac = nil
```

Comment:
`@time_parser_rfc5424_without_subseconds` ?

Reply:
Oh, I've fixed and pushed.